### ISSUE TITLE
Tool request: ALE generator

### ISSUE THREAD

**Issue by FJonske** [better-modernized-combat/bmod-client#27]
ALEs can be systematically created, as they are ostensibly particularly clunky XMLs.
The following features are intended:

- [ ] Alternate projectile ALEs for variants, all of which follow the same rules of increased or decreased size and color saturation.

- [ ] Randomized projectile generation (randomized number and style of emitters and particles) for use in Chaos mod or as a baseline for future weapons we didn't even think about.

This one is probably one for the backlog.

**Comment by FJonske** [comment]
XML can suck an egg and I'm moving the random generator to the backlog - making regular variants is fairly easy to automate (in fact, it works), but for various reasons its not really possible to make more intense projectiles without either blinding the player or making it look weird. Or investing a lot of manual work, which defeats the point.

For now, @IrateRedKite has made a muzzle lightning effect, which we will simply add to a couple of variants.

**Comment by IrateRedKite** [comment]
Fair enough! I've removed this from scope, I'll go ahead and get the VFX lines in for the extra muzzle flash on xlt variants if you just want to tweak the script to have the xlts take a different flash effect :)
